packages/client/src/client/App.tsx

```diff
@@ -23,6 +24,7 @@ export async function initApp() {
 
   registerFxNotifications()
```

Comment:
please move `registerFxNotifications` to `LiveRatesCore.tsx`
and squash Credit methods to one, and replace the quote reg in `CreditRfqsCore`